css/header.css

```diff
@@ -0,0 +1,58 @@
+@import url("https://fonts.googleapis.com/css2?family=PT+Sans&display=swap");
+
+* {
+    box-sizing: border-box;
+    font-family: "PT Sans", sans-serif;
+}
+
+.header {
+    padding: 1em 0;
+    padding-bottom: 0.7em;
+    position: fixed;
+    top: 0;
+    width: 100vw;
```

Comment:
add this code 
 1. left: 0;
 2. right: 0;
remove line number 13 - width: 100vh;
to see what it will do, add some background color to the header and then apply above code

Reply:
I see, ack, done